tools/preflight/preflight.sh

```diff
@@ -123,12 +129,12 @@ check_kubectl_access() {
   if [[ -n ${KUBECONFIG_PATH} ]]; then
     export KUBECONFIG=${KUBECONFIG_PATH}
   fi
-  echo -e "${LIGHT_BLUE}Checking access to the Kubernetes context $(kubectl config current-context)...${NC}\n"
+  echolog -e "${LIGHT_BLUE}Checking access to the Kubernetes context $(kubectl config current-context)...${NC}\n"
 
   if [[ $(kubectl get ns default) ]]; then
-    echo -e "${GREEN} ${CHECK} Able to access the default Kubernetes namespace${NC}\n"
+    echolog -e "${GREEN} ${CHECK} Able to access the default Kubernetes namespace${NC}\n"
```

Comment:
Will it be printed on stdout as well? Expectation is, such checks should be printed.